README.md

```diff
@@ -11,6 +11,29 @@ The main goal of this project is to provide a compatible, reliable and efficient
 
 The project is in early development.
 
+### Installation
+`OpenCombine` contains of two targets: `OpenCombine` and `OpenCombineDispatch`.
+
+##### Swift Package Manager
+To add `OpenCombine` to your [SPM](https://swift.org/package-manager/) package, add the `OpenCombine` package to the list of package and target dependencies in your `Package.swift` file.
```

Comment:
```suggestion
To add `OpenCombine` to your [SPM](https://swift.org/package-manager/) package, add the `OpenCombine` package to the list of packages and target dependencies in your `Package.swift` file.
```

I think we should also mention Xcode, as many users will probably want to use the SPM integration:

```markdown
You can add OpenCombine as an SPM dependency right in your Xcode project (requires Xcode 11).
For that use **File** → **Swift Packages** → **Add Package Dependency…**, enter the repository URL,
choose the latest available version, and don't forget to activate the checkboxes:
<img alt="Select the OpenCombine and OpenCombineDispatch targets" src="https://user-images.githubusercontent.com/16309982/67618468-bd379f80-f7f8-11e9-917f-e76e878a1aee.png">
```